docs/OV_Runtime_UG/Operations_specifications.md

```diff
@@ -5,193 +5,193 @@
+   ConvolutionBackpropData-1 <openvino_docs_ops_convolution_ConvolutionBackpropData_1>
+   Convolution-1 <openvino_docs_ops_convolution_Convolution_1>
+   Cos-1 <openvino_docs_ops_arithmetic_Cos_1>
+   Cosh-1 <openvino_docs_ops_arithmetic_Cosh_1>
+   CTCLoss-4 <openvino_docs_ops_sequence_CTCLoss_4>
+   CumSum-3 <openvino_docs_ops_arithmetic_CumSum_3>
+   DeformableConvolution-1 <openvino_docs_ops_convolution_DeformableConvolution_1>
+   DeformableConvolution-8 <openvino_docs_ops_convolution_DeformableConvolution_8>
+   DeformablePSROIPooling-1 <openvino_docs_ops_detection_DeformablePSROIPooling_1>
+   DepthToSpace-1 <openvino_docs_ops_movement_DepthToSpace_1>
+   DetectionOutput-1 <openvino_docs_ops_detection_DetectionOutput_1>
+   DetectionOutput-1 <openvino_docs_ops_detection_DetectionOutput_8>
+   Discrete Fourier Transformation (DFT)-7 <openvino_docs_ops_signals_DFT_7>
+   Divide-1 <openvino_docs_ops_arithmetic_Divide_1>
+   Einsum-7 <openvino_docs_ops_matrix_Einsum_7>
+   Elu-1 <openvino_docs_ops_activation_Elu_1>
+   EmbeddingBagOffsetsSum-3 <openvino_docs_ops_sparse_EmbeddingBagOffsetsSum_3>
+   EmbeddingBagPackedSum-3 <openvino_docs_ops_sparse_EmbeddingBagPackedSum_3>
+   EmbeddingSegmentsSum-3 <openvino_docs_ops_sparse_EmbeddingSegmentsSum_3>
+   Equal-1 <openvino_docs_ops_comparison_Equal_1>
+   Erf-1 <openvino_docs_ops_arithmetic_Erf_1>
+   Exp-1 <openvino_docs_ops_activation_Exp_1>
+   ExperimentalDetectronDetectionOutput-6 <openvino_docs_ops_detection_ExperimentalDetectronDetectionOutput_6>
+   ExperimentalDetectronGenerateProposalsSingleImage-6 <openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_6>
+   ExperimentalDetectronPriorGridGenerator-6 <openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6>
+   ExperimentalDetectronROIFeatureExtractor-6 <openvino_docs_ops_detection_ExperimentalDetectronROIFeatureExtractor_6>
+   ExperimentalDetectronTopKROIs-6 <openvino_docs_ops_sort_ExperimentalDetectronTopKROIs_6>
+   ExtractImagePatches-3 <openvino_docs_ops_movement_ExtractImagePatches_3>
+   FakeQuantize-1 <openvino_docs_ops_quantization_FakeQuantize_1>
+   FloorMod-1 <openvino_docs_ops_arithmetic_FloorMod_1>
+   Floor-1 <openvino_docs_ops_arithmetic_Floor_1>
+   GRN-1 <openvino_docs_ops_normalization_GRN_1>
+   GRUCell-3 <openvino_docs_ops_sequence_GRUCell_3>
+   GRUSequence-5 <openvino_docs_ops_sequence_GRUSequence_5>
+   GatherTree-1 <openvino_docs_ops_movement_GatherTree_1>
+   Gather-1 <openvino_docs_ops_movement_Gather_1>
+   Gather-7 <openvino_docs_ops_movement_Gather_7>
+   Gather-8 <openvino_docs_ops_movement_Gather_8>
+   GatherElements-6 <openvino_docs_ops_movement_GatherElements_6>
+   GatherND-5 <openvino_docs_ops_movement_GatherND_5>
+   GatherND-8 <openvino_docs_ops_movement_GatherND_8>
+   Gaussian Error Linear Unit (GELU)-2 <openvino_docs_ops_activation_GELU_2>
+   Gaussian Error Linear Unit (GELU)-7 <openvino_docs_ops_activation_GELU_7>
```

Comment:
Please use op type name.

Reply:
@andrew-zaytsev, are we building the list automatically? If not, please comment on possibility to generate it.